odoo/tests/common.py

```diff
@@ -810,9 +811,9 @@ def _wait_code_ok(self, code, timeout):
                     })
             elif res:
                 self._logger.debug('chrome devtools protocol event: %s', res)
-        self._logger.error('Script timeout exceeded : %s', (time.time() - start_time))
         self.take_screenshot()
-        return False
+        raise ChromeBrowserException(exception_details)('Script timeout exceeded : %s', (time.time() - start_time))
```

Comment:
I don't know what's supposed to happen, but it's probably not going to: exceptions are not callable, so this is going to blow up.

Reply:
Good catch